homeassistant/components/bring/coordinator.py

```diff
@@ -68,28 +72,21 @@ async def _async_update_data(self) -> dict[str, BringData]:
                 "Authentication failed but re-authentication was successful, trying again later"
             ) from e
 
-        list_dict: dict[str, BringData] = {}
-        for lst in lists_response["lists"]:
+        for lst in self.lists:
             try:
-                items = await self.bring.get_list(lst["listUuid"])
+                response = await self.bring.get_list(lst["listUuid"])
+                items.append(BringData(**response))
             except BringRequestException as e:
                 raise UpdateFailed(
                     "Unable to connect and retrieve data from bring"
                 ) from e
             except BringParseException as e:
                 raise UpdateFailed("Unable to parse response from bring") from e
-            else:
-                list_dict[lst["listUuid"]] = BringData(**lst, **items)
 
-        return list_dict
+        return items
```

Comment:
Refactor the coordinator to still return a dict with str and list, because then you don't have to cycle over it multiple times to get the list you want